docs/POLYFILLS.md

```diff
@@ -0,0 +1,117 @@
+# Using Polyfills
+
+In the case you would like to use Internet Explorer, you might be interested in polyfills. 
+We don't use them by default in our application, but we've created a way to do so in three easy steps. 
+
+## Creation of the file
+
+First, create an empty file called polyfills.browser.ts at the following location: 
+`src > polyfills.browser.ts`
+
+## Content of the file
+
+Second, copy/paste the following content into the file: 
```

Comment:
"The following polyfills are suitable for most of the cases, especially when targeting Internet Explorer:"